app/src/main/java/com/appunite/loudius/LoudiusApplication.kt

```diff
@@ -17,7 +17,35 @@
 package com.appunite.loudius
 
 import android.app.Application
+import com.appunite.loudius.di.dataSourceModule
+import com.appunite.loudius.di.dispatcherModule
+import com.appunite.loudius.di.githubHelperModule
+import com.appunite.loudius.di.networkModule
+import com.appunite.loudius.di.repositoryModule
+import com.appunite.loudius.di.serviceModule
+import com.appunite.loudius.di.viewModelModule
 import dagger.hilt.android.HiltAndroidApp
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.GlobalContext.startKoin
+import org.koin.dsl.module
 
 @HiltAndroidApp
```

Comment:
Question: Why do we still have hilt annotation?

Reply:
you are correct ;s not needed anymore